apps/designer/app/routes/designer/$id.tsx

```diff
@@ -35,17 +38,42 @@ type Error = {
 
 export const action: ActionFunction = async ({ request, params }) => {
   if (params.id === undefined) throw new Error("Project id undefined");
+  try {
+    if (request.method === "DELETE") {
+      const formData = await request.formData();
+      if (formData.get("assetId")) {
+        const id = formData.get("assetId") as string;
+        const name = formData.get("assetName") as string;
+        const deletedAsset = await deleteAsset({
+          id,
+          name,
+          dirname: __dirname,
```

Comment:
please don't use it, we need something importable from a single location for __dirname, I feel like I mentioned it before and wonder now if we have multiple places now

Reply:
Then i can't delete the file locally. Do you have a better option?